docs/finality-provider-operation.md

```diff
@@ -373,6 +376,27 @@ this value by specifying a custom address with the `--rpc-listener` flag.
 EOTS Manager Daemon is fully active!
 ```
 
+#### 5.3.1. Unlock file-based keyring
+
+⚠️ If you are using a `file` based keyring-backend, you need to unlock the keyring by executing the following command:
```

Comment:
When does someone need to unlock their keyring (e.g., after starting the daemon) and why (e.g., to be able to sign requests?)? If they don't do so, what happens (e.g., the program will error)?

After starting the daemon? And also in order to